docs/hardware.md

```diff
@@ -8,7 +8,10 @@ These are platforms that are currently commercially available for anybody to buy
 
 - Holybro:
    - [Holybro X500 V2 - PX4 Developer Kit](https://holybro.com/product/x500-v2-kit)
-   - [PX4 Vision Dev Kit V1.5](https://holybro.com/collections/multicopter-kit/products/px4-vision-dev-kit-v1-5) 
+   - DISCONTINUED [PX4 Vision Dev Kit V1.5](https://holybro.com/collections/multicopter-kit/products/px4-vision-dev-kit-v1-5)
```

Comment:
I did notice that this was sold out, but the website doesn't say it is discontinued? I'm figuring that it is though as it has been in this state for a while

Reply:
Yes probably a good idea untill they say it is discontinued.

As a possible future feature, we could make a table to indicate stocklevels. The dji tello was also out of stock for a while and then they suddenly discontinued it... so it can be a sign in some cases

Reply:
@knmcguire should we add some format to DISCONTINUED products?

Reply:
with format, do you mean a premade tag of some sorts? Or do you mean like make it italic?

Reply:
Yes, it can be Italic or using colors as well, or even inside a box, we only have to add that extra to a css and then declare it to mkdocs, what do you think?

Reply:
That is possible but that is perhaps worth a different PR. Also, I've seen that now there is actually a new section called 'Discontinued Platforms'. we can also place it there?

Reply:
Sure!